src/thread/thread_manager.cpp

```diff
@@ -658,7 +658,7 @@ namespace cubthread
 
 } // namespace cubthread
 
-void thread_initialize_manager (THREAD_ENTRY *thread_p)
+void thread_initialize_manager (THREAD_ENTRY *&thread_p)
```

Comment:
Is there really a reason for this function to exist? Can't just `cubthread::initialize (thread_p)` be called directly?

Reply:
Yes, there is an issue on Windows exporting the function from libcubrid.dll. Usually, these functions are added to the .def file, but namespaces are a problem. This was preferred to alternative solutions.